src/Raven.Studio/typescript/viewmodels/common/notificationCenter/detailViewer/operations/smugglerDatabaseDetails.ts

```diff
@@ -7,6 +7,7 @@ import generalUtils = require("common/generalUtils");
 import genericProgress = require("common/helpers/database/genericProgress");
 import delayBackupCommand = require("commands/database/tasks/delayBackupCommand");
 import viewHelpers = require("common/helpers/view/viewHelpers");
+import copyToClipboard = require("common/copyToClipboard");
```

Comment:
looks like prettier was executed on this file. Can you double check your configuration and if this file is ignored then revert this formatting to avoid merge hell in future?